Description: Disable lgi check at build time (only used at runtime)
 Another solution would be to AWESOME_IGNORE_LGI=1 at all targets, but
 it would cluttered unnecessarily debian/rules.
Forwarded: not-needed
Author: Arnaud Fontaine <arnau@debian.org>

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -102,10 +102,6 @@
     ${AWESOME_REQUIRED_LDFLAGS}
     ${AWESOME_OPTIONAL_LDFLAGS})
 
-# check for lgi and the needed gobject introspection files
-add_custom_target(lgi-check ALL
-    COMMAND ${SOURCE_DIR}/build-utils/lgi-check.sh)
-
 # atoms
 file(MAKE_DIRECTORY ${BUILD_DIR}/common)
 add_custom_command(
